delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/search/SearchParams.scala

```diff
@@ -119,4 +120,15 @@ object SearchParams {
         organization.forall(_ == resource.value.organizationLabel)
   }
 
+  final case class ResolverSearchParams(
+      deprecated: Option[Boolean] = None,
```

Comment:
Add `projectRef: Option[ProjectRef]`

Reply:
I don't think this is needed for the API; maybe just internally.

Reply:
I don't think there is any harm. Alternatively we need another method on `ResolverSearchParams`:
```scala
def matches(resource: ResourceF[Resolver], projectRef: ProjectRef): Boolean
```

but I think it is more a hussle.